crates/router/src/core/relay/utils.rs

```diff
@@ -32,7 +32,7 @@ pub async fn construct_relay_refund_router_data<'a, F>(
     let webhook_url = Some(payments::helpers::create_webhook_url(
         &state.base_url.clone(),
         merchant_id,
-        connector_name,
+        connector_account.merchant_connector_id.get_string_repr(),
```

Comment:
Can we have the function itself accept `&MerchantConnectorAccountId`? Update the usages of the function elsewhere as well.

Reply:
For reference, the same change had been taken up in #6631, but that PR seems to have gone stale.